### ISSUE TITLE
App shows different icons for the primary and the secondary switch

### ISSUE THREAD

**Issue by pz1** [maros/Zway-VisionZD2102#1]
the app shows different icons for the primary and the secondary switch (Zway v2.2.4)
![zd2102](https://cloud.githubusercontent.com/assets/7747627/19835156/1d2c4792-9e7e-11e6-8b87-e7107031b598.PNG)

**Comment by maros** [comment]
Well, the secondary input is displayed as a door sensor (probeType: general_purpose) which is correct afaict. The ternary input is triggered by tampering the device and is therefore using an 'alarm' icon (probeType: alarm_burglar). However, there seem to be differences between batches or minor revisions of the VisionZD2102 sensor. Newer devices seem to have this distinction between external sensor and tamper sensor, whereas older versions report everything (internal tamper switch and external sensor) as a tampering alarm.

**Comment by pz1** [comment]
These are the vDevs that ZWay generated for me.
![vision](https://cloud.githubusercontent.com/assets/7747627/19836933/c1a83b2a-9eae-11e6-89b1-8e522c7e662b.PNG)
I interpreted it that the "button" stands for the tamper switch, and the "Bovenraam" represents what I called primary switch. The secondary switch is the reed-relay  that I added. That one is displayed as I expect. So the problem imho is in how ZWay treats ZD2102.
Maybe something needs to be done about the icon/type of the primary switch with postfix sometime.
